packages/toolpad-app/src/components/Home.tsx

```diff
@@ -266,31 +294,23 @@ function AppCard({ app, activeDeployment, onDelete }: AppCardProps) {
             </IconButton>
           }
           disableTypography
-          title={
-            editingTitle ? (
-              <TextField
-                variant="standard"
-                size="small"
-                inputRef={appTitleInput}
-                sx={{ paddingBottom: '4px' }}
-                InputProps={{ sx: { fontSize: '1.5rem', height: '1.5em' } }}
-                onKeyUp={handleAppTitleInput}
-                onBlur={handleAppTitleBlur}
-                defaultValue={appTitle}
-              />
-            ) : (
-              <Typography gutterBottom variant="h5" component="div">
-                {app ? appTitle : <Skeleton />}
-              </Typography>
-            )
-          }
           subheader={
             <Typography variant="body2" color="text.secondary">
-              {app ? `Edited: ${app.editedAt.toLocaleString('short')}` : <Skeleton />}
+              {app ? `Edited ${getReadableDuration(app.editedAt)}` : <Skeleton />}
```

Comment:
We still need to make the absolute time accessible for the users as well. Perhaps using a `<time>` tag. Or with a `Tooltip`.